Tests/StorageTests/ZoomLevelStoreTests.swift

```diff
@@ -0,0 +1,94 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+@testable import Storage
+
+class ZoomLevelStoreTests: XCTestCase {
+    let zoomLevelStore = ZoomLevelStore.shared
+
+    let testHost1 = "www.example1.com"
+    let testHost2 = "www.example2.com"
+    let testHost3 = "www.example3.com"
+    let testZoomLevel: CGFloat = 2.0
+
+    override func tearDown() {
+        super.tearDown()
+        cleanUp()
+    }
```

Comment:
We should `nil` any instances we create for tests since each of the tests create individual instance and doesn't deinit them after completion. With a setUp and tearDown we ensure we properly clear any instances and start with a clean slate on each test case. 
```suggestion
    var zoomLevelStore: ZoomLevelStore!

    let testHost1 = "www.example1.com"
    let testHost2 = "www.example2.com"
    let testHost3 = "www.example3.com"
    let testZoomLevel: CGFloat = 2.0
    
    override func setUp() {
        super.setUp()
        zoomLevelStore = ZoomLevelStore.shared
        cleanUp()
    }

    override func tearDown() {
        super.tearDown()
        zoomLevelStore = nil
        cleanUp()
    }
```